linode/instanceconfig/schema_resource.go

```diff
@@ -61,9 +61,12 @@ var resourceSchema = map[string]*schema.Schema{
 		Description: "Helpers enabled when booting to this Linode Config.",
 	},
 	"interface": {
-		Type:        schema.TypeList,
-		Elem:        &schema.Resource{Schema: interfaceSchema},
-		Optional:    true,
+		Type:     schema.TypeList,
+		Elem:     &schema.Resource{Schema: interfaceSchema},
+		Optional: true,
+		Deprecated: "The embedded interface blocks in linode_instance_config is deprecated " +
+			"and scheduled to be remove in a future version. Please consider migrating to " +
+			"linode_instance_config_interface resource.",
```

Comment:
Should we add this deprecation message to the `interface` fields in `linode_instance` as well?

Reply:
We got that in `linode_instance` as well? LOL

Okay, let me add it.

Reply:
As discussed in DM, we decided to not to deprecate `interface` attribute in `linode_instance` for now because it's still supported by the API.